### ISSUE TITLE
pip dependency adns removed but still needed?

### ISSUE THREAD

**Issue by juju4** [CIRCL/AIL-framework#134]
Hello,

adns has been commented out in pip requirements
https://github.com/CIRCL/AIL-framework/commit/6b844d220238c71ca7beaeb094373a223b6071d3

but when doing install through my ansible role (https://github.com/juju4/ansible-ail-framework), it seems still needed
```
kitchen@81daf9206292:~$ tail -10 /var/log/supervisor/ail-script-Web.log 
    from cymru.ip2asn.dns import DNSClient as ip2asn
  File "/var/_ail/AILENV/local/lib/python2.7/site-packages/cymru/ip2asn/dns.py", line 13, in <module>
    import adns
ImportError: No module named adns
Traceback (most recent call last):
  File "/var/_ail/AIL/bin/Web.py", line 24, in <module>
    from cymru.ip2asn.dns import DNSClient as ip2asn
  File "/var/_ail/AILENV/local/lib/python2.7/site-packages/cymru/ip2asn/dns.py", line 13, in <module>
    import adns
ImportError: No module named adns

```
fail: https://travis-ci.org/juju4/ansible-ail-framework/builds/264166325

If I re-enable it, pass again https://travis-ci.org/juju4/ansible-ail-framework/builds/266343141

error or missing code?

Thanks

**Comment by adulau** [comment]
I thought adns was no more needed for DomainClassifier. But indeed, maybe the ip2asn still requires it. Good point.

**Comment by mokaddem** [comment]
Fixed in #135 
Thanks!
